tools/monorepo-utils/src/core/logger.ts

```diff
@@ -25,10 +25,20 @@ export class Logger {
 		] as number;
 	}
 
-	static error( message: string ) {
+	static error( err: unknown, failOnErr = true ) {
 		if ( Logger.loggingLevel >= LOGGING_LEVELS.error ) {
-			error( chalk.red( message ) );
-			process.exit( 1 );
+			if ( err instanceof Error ) {
+				error( chalk.red( err.message ) );
+			} else if ( typeof err === 'string' ) {
+				error( chalk.red( err ) );
+			} else {
+				// Best effort to log the error when we don't know the type.
+				error( chalk.red( JSON.stringify( err ) ) );
```

Comment:
If we know the error is an object and you want to stringify it, then passing in these extra arguments makes it readable:

```
console.log(JSON.stringify({foo:1, bar:2}, null, 4));
```
See https://stackoverflow.com/questions/18184559/how-to-pretty-print-log-output-in-chrome-devtools-console
And you can check in the console

Reply:
Oh yeah the indentation looks better that way, thanks!

Reply:
@psealock resolved in https://github.com/woocommerce/woocommerce/pull/38160/commits/4068828a08e924391db16d9079cd4deb2d1dcd0b

Note I just went with 2 spaces just to keep the output tighter for large objects. It still looks more readable. Thanks for the suggestion.